index.html

```diff
@@ -25,95 +25,333 @@
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <img
+            class="second-img"
+            src="./circle_header.svg"
+            alt="circle 2 header"
+          />
 
-      <!-- PAGE BREAK -->
-      <section class="page-break">
-        <div>
-          <p>Work &darr;</p>
+          <hr class="first-line" />
+
+          <!-- TECHNLOGIES -->
+          <section class="technologies">
+            <div class="technologies-item">
+              <div class="technologies-title">Languages</div>
+              <ul>
+                <li>JavaScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+              </ul>
+            </div>
+            <div>
+              <div class="technologies-item">
+                <div class="technologies-title">Frameworks</div>
+                <ul>
+                  <li>BootStrap</li>
+                  <li>Ruby on Rails</li>
+                  <li>RPSec</li>
+                  <li>CapyBase</li>
+                  <li>Selenium</li>
+                </ul>
+              </div>
+            </div>
+            <div>
+              <div class="technologies-item">
+                <div class="technologies-title">Skills</div>
+                <ul>
+                  <li>Codekit</li>
+                  <li>GitHub</li>
+                  <li>CodePen</li>
+                  <li>GitLab</li>
+                  <li>Terminal</li>
+                </ul>
+              </div>
+            </div>
+          </section>
+          <hr class="first-line" />
+
+          <!-- PAGE BREAK -->
+          <section class="page-break">
+            <div>
+              <p>Work &darr;</p>
+            </div>
+            <div class="social-links">
+              <ul>
+                <li>
+                  <a href="https://github.com/Marlyn_Mayienga"
+                    ><i class="fab fa-github"></i
+                  ></a>
+                </li>
+                <li>
+                  <a href="https://www.linkedin.com/in/marlyn-mayienga/"
+                    ><i class="fab fa-linkedin"></i
+                  ></a>
+                </li>
+                <li>
+                  <a href="https://www.instagram.com/merl.__/?hl=en"
+                    ><i class="fas fa-cat"></i
+                  ></a>
+                </li>
+                <li>
+                  <a href="https://twitter.com/M_ayienga"
+                    ><i class="fab fa-twitter"></i
+                  ></a>
+                </li>
+                <li>
+                  <a href="marlymayienga@gmail.com"
+                    ><i class="fas fa-envelope"></i
+                  ></a>
+                </li>
+              </ul>
+            </div>
+            <div>
+              <p>&darr; Work</p>
+            </div>
+          </section>
```

Comment:
- [ ] Kindly change the color for the paragraph and the technologies to `#B3BAC5;` to match the Figma design.
- [ ] Kindy make the two buttons `Work` a little bit away from the accounts icons.

Figma design
![image](https://user-images.githubusercontent.com/35535130/166837929-8263f158-dfdc-4046-86c1-00591433e282.png)


Your design
![image](https://user-images.githubusercontent.com/35535130/166837892-fbea03ab-7e86-4483-8157-a1f17c7c5748.png)